website/pages/blog/announcing-graphql-network-inspector.mdx

```diff
@@ -0,0 +1,77 @@
+image: /blog-assets/announcing-graphql-network-inspector/cover.png
+thumbnail: /blog-assets/announcing-graphql-network-inspector/thumbnail.png
+---
+
+GraphQL Network Inspector is a dedicated devtools extension which allows you to inspect and debug
+GraphQL queries and mutations from network traffic.
+
+The extension has seen great adoption since its initial release and now I'm excited to announce it's
+new home as part of The Guild.
+
+You can check out the open source extension and try it out today.
+[GraphQL Network Inspector](https://github.com/warrenday/graphql-network-inspector)
+
+## Why GraphQL Network Inspector?
+
+If you've ever worked with GraphQL on the client, you'll know that it can be a real pain to debug
+using the standard network tab. The standard devtools are designed for REST APIs and don't provide
+clear visibility into the individual GraphQL queries and mutations being sent.
+
+With every request you'll see `POST /graphql`. This makes it difficult to distinguish requests
+without opening the request details and inspecting the body.
+
+GraphQL Network Inspector solves this problem by providing a dedicated tab for GraphQL requests and
+displaying each query in a readable format.
+
+![Compare](/blog-assets/announcing-graphql-network-inspector/request-compare.png)
+
+## Features Overview
+
+The great thing about a dedicated inspector is we can provide much more specific functionality
+around GraphQL. Here are some of the features we've added to the extension:
+
+### Request Overview
+
+The request overview tab provides a simple overview of all queries and mutations. Here you can see
+queries by name, distinguish between queries and mutations and even see total errors returned for
+each request.
+
+![Request Overview](/blog-assets/announcing-graphql-network-inspector/request-overview.png)
+
+### Request Details
+
+Clicking into a request will show similar details to the standard inspector. But here we have a
+better breakdown for each element of the request including; headers, request query with variables
+and response.
+
+![Request Details](/blog-assets/announcing-graphql-network-inspector/request-details.png)
+
+### Copy Details to GraphQL Playground
+
+The extension also provides convenient buttons to copy the request query and variables into GraphQL
+Playground, great if you want to re-run the query.
+
+Under the request details tab, you'll see a button to copy the query and variables to the clipboard.
+
+You can also click individual headers to copy them as well.
+
+![Copy to Playground](/blog-assets/announcing-graphql-network-inspector/copy-header.png)
+
+### Convenient Features
+
+Streamlining the debugging process is a key goal of the extension. On top of the features above
+we've also introduces helpful extras such as full-text search and filtering.
+
+![Copy to Playground](/blog-assets/announcing-graphql-network-inspector/search.png)
+
+So check it out today and don't forget to give
```

Comment:
I would also ask what other features they would want and if they would like to see integrations with other Guild and ecosystem tooling?